src/app/SBE/table/HeaderCell.js

```diff
@@ -1,30 +1,36 @@
-import { replace } from 'ramda'
+import { isEmpty, replace } from 'ramda'
 import { HStack, IconButton, Text, Th } from '@chakra-ui/react'
 import { useSelector } from 'react-redux'
 import { selectCode } from 'redux/db/selectors'
-import { onSendAnswer } from 'vertx'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faAngleDown, faAngleUp } from '@fortawesome/free-solid-svg-icons'
+import sendEvtClick from 'app/ASKS/utils/send-evt-click'
 
 const Cell = ({ attribute, parentCode }) => {
+  const userCode = useSelector(selectCode('USER'))
   const data = useSelector(selectCode(parentCode, attribute))
   const sort = useSelector(selectCode(parentCode, replace('COL_', 'SRT_', attribute))) || {}
 
+  const onClick = () => {
+    sendEvtClick({
+      processId: 'no-idq', //Not sure this is great but I don't know if we'll get this from anywhere
+      attributeCode: sort.attributeCode,
+      code: 'ACT_SRT',
```

Comment:
Can we get the code from the backend?

Reply:
Bryn has asked that I send `ACT_SRT` hardcoded, but we could try to include it in the SBE somewhere in the future